docs/components/codeblock.md

```diff
@@ -217,6 +217,40 @@ Sets the display theme of the component.
 />
 ```
 
+### isSingleLine
+
+* **Type**: `boolean`
+* **Required**: no
+* **Default**: `false`
+
+The code passed into the component will be displayed on a single line.
+
+::: tip NOTE
+Please keep the following in mind when using `isSingleLine`:
+- Line numbers will not be displayed
+- Code will not be searchable
+- Long code snippets will be truncated to allow for responsive design
```

Comment:
I'm still wondering if we should instead allow the code to overflow:auto horizontally and scroll... I'll think about this when the deploy preview builds so I can play with it 😬

Reply:
Definitely a good option to consider, let me know what you think and I can make any changes